xml/System.Runtime.CompilerServices/TaskAwaiter.xml

```diff
@@ -175,14 +161,7 @@
         <summary>Gets a value that indicates whether the asynchronous task has completed.</summary>
         <value>
           <see langword="true" /> if the task has completed; otherwise, <see langword="false" />.</value>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- This type and its members are intended for use by the compiler.  
-  
- ]]></format>
-        </remarks>
+        <remarks />
```

Comment:
It's unclear whether this property should have the wording removed based on Stephen's remark. Do you think it should be removed?

Reply:
@gewarren I don't think this remark should be removed. How to 'un-do' my change?